modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -15,27 +16,32 @@ Restic is not supported in the OADP on ROSA with AWS STS environment. Ensure the
 
 .Prerequisites
 
-* A ROSA OpenShift Cluster with the required access and tokens.
-* link:https://docs.openshift.com/container-platform/4.14/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#oadp-creating-default-secret_installing-oadp-aws[A default Secret], if your backup and snapshot locations use the same credentials, or if you do not require a snapshot location.
+* A ROSA OpenShift Cluster with the required access and tokens. For instructions, see the procedure in _Prepare AWS credentials_.
+
+
+* link:https://docs.openshift.com/container-platform/4.13/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#oadp-creating-default-secret_installing-oadp-aws[A default Secret], if your backup and snapshot locations use the same credentials, or if you do not require a snapshot location.
```

Comment:
I think this was just copied over from other OADP install docs.
+1 this specific rosa sts do not have a prerequisite of a default secret since it's shown in the next steps on this page already.

Reply:
removed from prerequisite